app/src/main/java/com/greenfox/opal/gitinder/fragments/SettingsFragment.java

```diff
@@ -1,22 +1,72 @@
 package com.greenfox.opal.gitinder.fragments;
 
+import android.content.Context;
+import android.content.SharedPreferences;
 import android.os.Bundle;
 import android.support.annotation.Nullable;
 import android.support.v4.app.Fragment;
 import android.util.Log;
 import android.view.LayoutInflater;
 import android.view.View;
+import android.view.View.OnClickListener;
 import android.view.ViewGroup;
+import android.widget.Switch;
+import android.widget.Toast;
 import com.greenfox.opal.gitinder.R;
+import javax.inject.Inject;
 
 public class SettingsFragment extends Fragment {
 
   private static final String TAG = "SettingsFragment";
 
+  Switch switchNotifications;
+  Switch switchSync;
+  @Inject
+  SharedPreferences preferences;
+  SharedPreferences.Editor editor;
```

Comment:
@balintvecsey order of fields should be constants, dependencies, views, other fields